runtime-light/coroutine/runtime-future.h

```diff
@@ -0,0 +1,58 @@
+// Compiler for PHP (aka KPHP)
+// Copyright (c) 2024 LLC «V Kontakte»
+// Distributed under the GPL v3 License, see LICENSE.notice.txt
+
+#pragma once
+
+#include <cinttypes>
+#include <variant>
+
+struct fork_future {
+  int64_t awaited_fork_id;
+  uint64_t timeout_d;
+
+  explicit fork_future(int64_t fork_id)
+    : awaited_fork_id(fork_id)
+    , timeout_d(0) {}
+  fork_future()
+    : awaited_fork_id(-1)
+    , timeout_d(0) {}
+};
+
+struct stream_future {
+  uint64_t stream_d;
+  uint64_t timeout_d;
+
+  explicit stream_future(uint64_t stream)
+    : stream_d(stream)
+    , timeout_d(0) {}
+  stream_future()
+    : stream_d(0)
+    , timeout_d(0) {}
+};
+
+namespace std {
+template<>
+struct hash<fork_future> {
+  std::size_t operator()(const fork_future &f) const {
```

Comment:
Let's not mix types with `std::` prefix and without it. We don't use that prefix in the project